checkov/terraform/checks/resource/aws/RedshiftServerlessNamespaceKMSKey.py

```diff
@@ -0,0 +1,21 @@
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories
+from checkov.common.models.consts import ANY_VALUE
+
+
+class RedshiftServerlessNamespaceKMSKey(BaseResourceValueCheck):
+    def __init__(self) -> None:
+        name = "Ensure that Redshift serverless namespace is encrypted by KMS"
+        id = "CKV_AWS_282"
+        supported_resources = ['aws_redshiftserverless_namespace']
+        categories = [CheckCategories.ENCRYPTION]
```

Comment:
```suggestion
        supported_resources = ('aws_redshiftserverless_namespace',)
        categories = (CheckCategories.ENCRYPTION,)
```
for static collections a `tuple` is always the better choice compared to a `list` 😄